### PR TITLE
feat: add build and deploy windows zip file support on Azure Stack

### LINKED ISSUE #1887: Add windows K8s build and deploy support for Azure Stack
Add windows K8s build and deploy support for Azure Stack.

### PR DESCRIPTION
**Pull request by rjaini**
**Reason for Change**:

Add build and deploy windows zip file support on Azure Stack

**Issue Fixed**:

Fixes #1887 

**Requirements**:
- [X] uses [conventional commit messages](https://www.conventionalcommits.org/)
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -94,7 +94,7 @@ build-cross:
 
 .PHONY: build-windows-k8s
 build-windows-k8s:
-	./scripts/build-windows-k8s.sh -v $(K8S_VERSION) -p $(PATCH_VERSION)
+	./scripts/build-windows-k8s.sh -v $(K8S_VERSION) -p $(PATCH_VERSION) -a $(BUILD_AZURE_STACK)
```
[the remark this answers is not in the document]
Comment (jadarsie) [reply Makefile]: or add a new target to build the azure stack code base

[the remark this answers is not in the document]
Comment (rjaini) [reply Makefile]: Added a new target to build azs code base.

[the remark this answers is not in the document]
Comment (jadarsie) [reply Makefile]: `build-windows-k8s` is repeated

**Pull request merged**